### ISSUE TITLE
ModelBuilder- Fail to Install Runtime Error

### ISSUE THREAD

**Issue by SeerIncLimited** [dotnet/machinelearning-modelbuilder#2993]
- Model Builder Version : 17.19.1.2460201
 - Visual Studio Version:17.12.4

**Bug Description: **
Model Builder throws a "Fail to install runtime error when the Train button is clicked. 
We note the error only occurs when the folder to which the file tensorflow.gpu.pdb file has an apostrophe in it.

**Steps to reproduce the behavior:**
1. Create a VS2022 Class Library project
2. Add a Machine Learning Model  (.mbconfig) to the project and configure as follows:
3. Select the Image Classification Scenario
4. Select Local GPU as the Environment and confirm GPU Compatibility
5. Add Data and check the Data Preview
6. Commence Training by click on the Train button

**Expected behavior:**
We would expect training to commence and complete with the Traing Complete tick and a Training Results summary.
However, we get the following error: PLEASE NOTE THE APOSTROPHE IN THE USERS FOLDER NAME

**Error:**
restore "c:\program files\microsoft visual studio\2022\professional\common7\ide\extensions\pee5r3yg.zb0\AutoMLService\RuntimeManager\tensorflow.gpu.csproj" --configfile "c:\program files\microsoft visual studio\2022\professional\common7\ide\extensions\pee5r3yg.zb0\AutoMLService\RuntimeManager\NuGet.config" -r win-x64 /p:UsingToolXliff=false /p:TorchSharpVersion=0.103.1 /p:TorchSharpCudaRuntimeVersion=2.4.0 /p:TensorflowRuntimeVersion=2.3.1 /p:BaseIntermediateOutputPath="C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1\obj"
publish "c:\program files\microsoft visual studio\2022\professional\common7\ide\extensions\pee5r3yg.zb0\AutoMLService\RuntimeManager\tensorflow.gpu.csproj" -r win-x64 -c Release --no-self-contained -o "C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1" --no-restore /p:UsingToolXliff=false /p:TorchSharpVersion=0.103.1 /p:TorchSharpCudaRuntimeVersion=2.4.0 /p:TensorflowRuntimeVersion=2.3.1 /p:BaseOutputPath="C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1\bin\\" /p:BaseIntermediateOutputPath="C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1\obj\\"
start installing runtime in C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1
  Determining projects to restore...
  All projects are up-to-date for restore.
  tensorflow.gpu -> C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1\bin\Release\netstandard2.0\win-x64\tensorflow.gpu.dll
C:\Program Files\dotnet\sdk\9.0.102\Microsoft.Common.CurrentVersion.targets(4925,5): error MSB3021: Unable to copy file "C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1\obj\Release\netstandard2.0\win-x64\tensorflow.gpu.pdb" to "@(_DebugSymbolsIntermediatePath->'C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1\bin\Release\netstandard2.0\win-x64\%(Filename)%(Extension)')". The filename, directory name, or volume label syntax is incorrect. : 'c:\program files\microsoft visual studio\2022\professional\common7\ide\extensions\pee5r3yg.zb0\AutoMLService\RuntimeManager\@(_DebugSymbolsIntermediatePath->'C:\Users\Bruce D'Angello\AppData\Local\ModelBuilder\tensorflow-cuda.2.3.1\bin\Release\netstandard2.0\win-x64'. [c:\program files\microsoft visual studio\2022\professional\common7\ide\extensions\pee5r3yg.zb0\AutoMLService\RuntimeManager\tensorflow.gpu.csproj]

**Screenshots:**
![Image](https://github.com/user-attachments/assets/2bcb72ae-711d-415a-a70f-61af0d07b473)

![Image](https://github.com/user-attachments/assets/c21f5eab-93f3-4319-9eb3-20dcc260e1ad)

![Image](https://github.com/user-attachments/assets/fd771e45-8c8f-47f3-bfdd-b7a5f22eb2c2)

![Image](https://github.com/user-attachments/assets/c419ff89-8f81-4507-a088-fa0b85926c72)

**Additional context:**
As mentioned running the running the project with no apostrophe in the logged on user name works successfully. 
Is there a way we can configure the output path.
